Mathlib/Data/Nat/Defs.lean

```diff
@@ -151,20 +151,10 @@ lemma le_of_pred_lt : ∀ {m}, pred m < n → m ≤ n
 
 lemma lt_iff_add_one_le : m < n ↔ m + 1 ≤ n := by rw [succ_le_iff]
 #align nat.lt_iff_add_one_le Nat.lt_iff_add_one_le
-
--- Just a restatement of `Nat.lt_succ_iff` using `+1`.
-lemma lt_add_one_iff : m < n + 1 ↔ m ≤ n := Nat.lt_succ_iff
-#align nat.lt_add_one_iff Nat.lt_add_one_iff
-
 -- A flipped version of `lt_add_one_iff`.
 lemma lt_one_add_iff : m < 1 + n ↔ m ≤ n := by simp only [Nat.add_comm, Nat.lt_succ_iff]
 #align nat.lt_one_add_iff Nat.lt_one_add_iff
 
--- This is true reflexively, by the definition of `≤` on ℕ,
--- but it's still useful to have, to convince Lean to change the syntactic type.
-lemma add_one_le_iff : m + 1 ≤ n ↔ m < n := Iff.rfl
-#align nat.add_one_le_iff Nat.add_one_le_iff
```

Comment:
And this

Reply:
I am really looking forward to these being gone!